src/Controller/SettingController.php

```diff
@@ -77,7 +77,22 @@ public function reRunAction(Request $request)
             $successRerun = true;
             $data = $reRunForm->getData();
             foreach ($data["batch"] as $command) {
-                shell_exec("php ../bin/console $command >/dev/null 2>&1 &");
+
+                $date = \DateTime::createFromFormat("d/m/Y", $data["date"]);
+
+                if ($command == "vendor:process" && $date) {
+                    // vendor:process 
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the useless trailing whitespaces at the end of this line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://hook.hipay.org:8000/sonar/hipay_sonar_user/dw8ATD37F8bmHSys/coding_rules#rule_key=php%3AS1131)